source/blog/2014-06-07-jquery-uk-conference-2014.html.markdown

```diff
@@ -54,7 +54,7 @@ In summary, conferences are awesome because...
 
 1) You get to hang out with your workmates
 
-![Pubbage in Oxford](https://fbcdn-sphotos-e-a.akamaihd.net/hphotos-ak-frc3/t1.0-9/10314527_10152491661949203_7440674471769565980_n.jpg)
```

Comment:
This image seems broken, we've copied over a "Access denied" graphic.  Might be worth talking to Tom to see if he knows where the original is, and we can copy that over properly.